README.md

```diff
@@ -50,7 +50,7 @@ The [Kotlin grammar reference](https://kotlinlang.org/docs/reference/grammar.htm
 
 ### Templates
 
-The Kotlin website uses [Jinja2](http://jinja.pocoo.org/docs/dev/) templates from the [templates](templates) directory.
+The Kotlin website uses [Jinja2](https://jinja.palletsprojects.com/en/2.11.x/) templates from the [templates](templates) directory.
```

Comment:
Not quite sure about the specified version?
@nikpachoo maybe it's better to specify the latest here? `https://jinja.palletsprojects.com/en/latest/` ?

Reply:
We don’t use the latest version; the link to 2.11.x is fine.